moveit_core/robot_state/test/robot_state_test.cpp

```diff
@@ -34,7 +34,6 @@
 
 /* Author: Ioan Sucan */
 #include <moveit_resources/config.h>
-#include <moveit/rdf_loader/rdf_loader.h>
```

Comment:
i think it would be nice if we moved the rdf_loader into moveit_core, since it only depends on:

#include <moveit/macros/class_forward.h>
#include <urdf/model.h>
#include <srdfdom/model.h>
#include <ros/ros.h>
#include <ros/package.h>

however the last two dependencies would be a big shift (that im a fan of)

we need better tools for testing code in moveit and this would be a big help

Reply:
+1 for better testing tools.